docs/Cluster-VictoriaMetrics.md

```diff
@@ -793,6 +793,14 @@ Some workloads may need fine-grained resource usage limits. In these cases the f
   when the database contains big number of unique time series because of [high churn rate](https://docs.victoriametrics.com/faq/#what-is-high-churn-rate).
   In this case it might be useful to set the `-search.maxLabelsAPIDuration` to quite low value in order to limit CPU and memory usage.
   See also `-search.maxLabelsAPISeries` and `-search.ignoreExtraFiltersAtLabelsAPI`.
+- `-search.maxFederateSeries` at `vmselect` limits maximum number of time series, which can be returned via [/federate API](https://docs.victoriametrics.com#federation).
+  The duration of the `/federate` queries is limited via `-search.maxQueryDuration` flag. This option allows limiting memory usage.
+- `-search.maxExportSeries` at `vmselect` limits maximum number of time series, which can be returned from [/api/v1/export* APIs](https://docs.victoriametrics.com#how-to-export-data-in-json-line-format).
+  The duration of the export queries is limited via `-search.maxExportDuration` flag. This option allows limiting memory usage.
+- `-search.maxTSDBStatusSeries` at `vmselect` limits maximum number of time series, which can be processed during the call to [/api/v1/status/tsdb](https://docs.victoriametrics.com#tsdb-stats).
+  The duration of the status queries is limited via `-search.maxStatusRequestDuration` flag. This option allows limiting memory usage.
+- `-search.maxDeleteSeries` at `vmselect` limits maximum number of time series, which can be deleted using [/api/v1/admin/tsdb/delete_series](https://docs.victoriametrics.com#how-to-delete-time-series).
```

Comment:
`-search.maxDeleteSeries` is already mentioned in L761
https://github.com/VictoriaMetrics/VictoriaMetrics/blob/e6b7d25ab44a3c8b975943ffa2b08efb91698b01/docs/Cluster-VictoriaMetrics.md?plain=1#L761